src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,220 @@
+The AWSCloudWatchLoggingPlugin log levels can be configured remotely.  When configured, the Amplify Logger will fetch the remote log levels, defined in a `remoteloggingconstraints.json` file and deployed to an S3 bucket, via an API Gateway endpoint that is integrated with a lambda.  The log levels defined in the remote logging constraints file will overwrite the local log levels and will persist across app sessions.
```

Comment:
You can remotely configure the Amplify Logger, enabling you to make changes to your logging levels or user allow list in your deployed applications.

Callout:
The logging configurations you set remotely will overwrite the local log level and persist for future app sessions.

Reply:
Updated the description and adde Callout.